core/src/main/java/org/infinispan/scattered/package-info.java

```diff
@@ -0,0 +1,228 @@
+/**
+ * <h1>SCATTERED CACHE DESIGN</h1>
```

Comment:
Please start the review here, this is updated version of https://github.com/infinispan/infinispan/wiki/Scattered-Cache-design-doc